javascripts/vjbug.js

```diff
@@ -1,6 +1,8 @@
 app.vjbug = {
   processDate: function () {
     var eventDate = document.getElementById('hiddenDate');
+    if(eventDate == null)
```

Comment:
Spacing here isn't correct and you should probably do

`if (typeof eventData === 'undefined')`